### PR TITLE
Cpu data

### PR DESCRIPTION
**Pull request by giulianogiari**

### CODE REVIEW

File: src/utils/cpu.py [hunk 1]
```diff
@@ -34,12 +41,46 @@ def get_cpu_usage(seconds):
...
+                data = json.load(open(amd_fname))
+            # Iterate through the json to get the model name and the TDP
+            for model_info in data['data']:
+                # remove non alphanumeric characters (i.e., TM) from Model name
+                model = re.sub(r'\W+', ' ', str(model_info['Model']))
+                if model in cpu_name:
+                    print(model)
+                    # get the TDP, remove the non numeric characters and convert to float
+                    tdp = float(re.sub(r'\D+', '', model_info['Default TDP']))
+                    break
```
Comment (FrnRmn) [inline src/utils/cpu.py]: I would change that find algorithm [line 72-79] with something that assures you that the order in which you iterate does not influence the outcome

**Review by FrnRmn — CHANGES_REQUESTED** [review]
Thanks for the update. In addition to the specific code line comment, I would try to match the personal CPU model with the database model using the alphanumeric code, as we discussed before!

File: src/utils/cpu.py (hunk 1 above)
Comment (MattiaSilvestri) [reply src/utils/cpu.py]: > 

You mean that models deep down in the list will require more time to be located?

**Review by MattiaSilvestri — COMMENTED** [review]
Good, only minor thing is that there are no annotations for the types in input to the functions, something like: `get_cpu_tdp(cpu_name: str) -> float:`
But I'm just being picky, it's not essential.

File: src/utils/cpu.py [hunk 2]
```diff
@@ -1,7 +1,13 @@
 import cpuinfo
+import json
+import os
 import psutil
+import re
+import warnings
```
Comment (FrnRmn) [inline src/utils/cpu.py]: It seems warnings is not used in the code

**Comment by FrnRmn** [comment]
Thanks for the updates, now it looks fine to me

**Review by MattiaSilvestri — APPROVED** [review]
Good to merge for me.

**Pull request merged**